flow/connectors/postgres/client.go

```diff
@@ -443,24 +443,21 @@ func (c *PostgresConnector) GetLastSyncBatchID(jobName string) (int64, error) {
 	return result.Int64, nil
 }
 
-func (c *PostgresConnector) GetLastSyncAndNormalizeBatchID(jobName string) (*model.SyncAndNormalizeBatchID, error) {
-	var syncResult, normalizeResult pgtype.Int8
+func (c *PostgresConnector) GetLastNormalizeBatchID(jobName string) (int64, error) {
+	var syncBatchID pgtype.Int8
```

Comment:
`normalizeBatchID`?

Reply:
renamed to `result` matching `GetLastSyncBatchID`